Sorting/ItemSorter.cs

```diff
@@ -186,11 +243,11 @@ public static List<Item> Aggregate(AggregateContext context, bool actuallyAggreg
 
 		private static IEnumerable<string> GetItemTooltipLines(Item item) {
```

Comment:
unnecessary `var` usage changes in this method.  please use the existing code in MagicStorage/1.4.4 as a comparison